src/detection/detection-card/content/items/attachements.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { Box, Typography } from '@material-ui/core'
+import DescriptionIcon from '@material-ui/icons/Description'
+
+import { useStyles } from 'detection/detection-card/content/items/items-style'
+
+export const Attachements = () => <Box className={useStyles().contentItem}>
```

Comment:
I'm pretty sure that's not the way to spell `attachments` :smile:

Reply:
done